impress-org/givewp
# Donors without WP accounts are now able to upload avatar images

_**NOTE: This PR is dependent on changes found in this PR: https://github.com/impress-org/givewp/pull/5798**_

Resolves #5743 

## Description

This PR resolves a bug where Donors without an associated WP account were not able to upload avatar images successfully. The issue is resolved through the introduction of a new "Avatar" route, which acts in place of the existing WP media route, and can handle GiveWP verified requests without requiring that Donors have a registered WP account.

## Affects

This PR affects backend logic for the Donor Dashboard application, specifically how avatar uploads are handled.

## Visuals

N/A

## Testing Instructions

1. Go to the Donor Dashboard
2. Log In with a Donor account (without an associated WP account)
3. Upload a new avatar
4. Click update, see that the spinner resolves and the update is succesful

## Pre-review Checklist
